examples/search_examples/public/components/app.tsx

```diff
@@ -117,6 +122,16 @@ export const SearchExamplesApp = ({
     setSelectedField(fields?.length ? fields[0] : null);
   }, [indexPattern]);
 
+  useEffect(() => {
+    const { sessionId } = getQueryParams(window.location as Location);
```

Comment:
typescript is failing here. 

`getQueryParams(createBrowserHistory().location)`

should work
https://github.com/elastic/kibana/pull/83073/files#diff-9469ecdf04195e710d0de657563f35a4f16a80f2d5839abfb22edc3f90c16b1bR130